plugin/pkg/admission/securitycontext/scdeny/admission.go

```diff
@@ -45,6 +46,11 @@ var _ admission.ValidationInterface = &Plugin{}
 
 // NewSecurityContextDeny creates a new instance of the SecurityContextDeny admission controller
 func NewSecurityContextDeny() *Plugin {
+	// DEPRECATED: SecurityContextDeny will be removed in favor of PodSecurity admission.
+	klog.Warningf("%s admission controller is deprecated. "+
+		"Please remove this controller from your configuration files and scripts. "+
+		"See https://k8s.io/docs/reference/access-authn-authz/admission-controllers/#securitycontextdeny for more information.",
```

Comment:
We should update https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#securitycontextdeny to explicitly say that the admission plugin is deprecated.

Reply:
Ok I already updated that doc here https://github.com/kubernetes/website/pull/39131 😅 but you mention a big "deprecation" warning I guess.